lib/services/auth.dart

```diff
@@ -32,6 +33,52 @@ class AuthService {
     }
   }
 
+  Future<bool> updateAccount(String email, String firstName, String lastName, String phone) async {
+    Map<String, String> requestBody = new Map<String, String>();
+
+    final storage = new FlutterSecureStorage();
+    String token = await storage.read(key: 'auth_token');
+
+    requestBody['email'] = email;
+    requestBody['first_name'] = firstName;
+    requestBody['last_name'] = lastName;
+    requestBody['phone'] = phone;
+
+    if (token != null) {
+      final response = await http.patch(
+          'https://claytoncornett.tk/api/user/',
+          headers: {HttpHeaders.authorizationHeader: "Token $token"}
+      );
+      print(response.statusCode);
+      final responseBody = jsonDecode(response.body);
+      print(requestBody);
+      if (response.statusCode == 200) {
+        //await AccountRepository.getInstance().getAccount();
+//        await AccountRepository.getInstance().invalidateCurrentAccount();
+        storage.write(key: 'auth_token', value: token);
```

Comment:
you don't need to write the token into storage since you're just reading it to make the PATCH call